docs/release-notes.md

```diff
@@ -14,4 +14,28 @@
   </tr>
 </table>
 
-# Release notes
\ No newline at end of file
+# Release notes
+
+## Trino Gateway 3 (xx Sep 2023)
+
+The first release of Trino Gateway is based on the Presto Gateway 1.9.5 codebase
```

Comment:
Should we link to https://github.com/lyft/presto-gateway to disambiguate which Presto Gateway we're based on?

Reply:
Is there any others?

Reply:
Anyway.. yes.. lets link .. should we say Presto Gateway from Lyft ?